template/php/template.yml

```diff
@@ -0,0 +1,2 @@
+language: php
+fprocess: php index.php
```

Comment:
In this file you can specify a welcome message explaining how to manage dependencies and which file to edit. See the others as examples.

Reply:
I'd keep the message here brief and simple, and add more details into the `php-extension.sh`, something like:

```
welcome_message: |
  You have created a new function which uses PHP 7.2.
  Dependencies and extensions can be added using the composer.yml
  and php-extension.sh files.
```